test/unit/org/apache/cassandra/index/sai/cql/StorageAttachedIndexDDLTest.java

```diff
@@ -285,11 +335,10 @@ public void shouldCreateIndexOnReversedType() throws Throwable
     }
 
     @Test
-    public void shouldCreateIndexWithAlias()
+    public void shouldCreateIndexWithFullClassName()
     {
         createTable("CREATE TABLE %s (id text PRIMARY KEY, val text)");
-
-        createIndex("CREATE CUSTOM INDEX ON %s(val) USING 'StorageAttachedIndex'");
+        createIndex("CREATE CUSTOM INDEX ON %s(val) USING 'org.apache.cassandra.index.sai.StorageAttachedIndex'");
```

Comment:
USING org.apache.cassandra.index.sai.StorageAttachedIndex also passed the test ,So the user document should be updated.

Reply:
Yes, if we suggest users to use a certain method, at least a document description or some warning is required. In my experience, if not, users either cannot use it or use it blindly.

In the end, this experience was terrible

Reply:
As far as I am concerned, I have participated  in a little discussion  with the previous DISSCUSS and had a look at this patch  once before, so I have a rough idea of which usage is feasible. For first-time users, the process may not be as smooth.

Reply:
Once we settle on a name for the legacy 2i and update that in the YAML and `Config`, it would be useful to update `CQL.textile` and `indexes.adoc` (which is the source for the OSS website docs) to reflect the recommended usage. I'll have another commit that includes that shortly...

Reply:
ok~~

Reply:
Pushed. Even this is just to illustrate basic supported usage. The SAI `README.md` in `o.a.c.index.sai` is probably a better in-tree overview of what SAI actually is.